src/lib/containers/QueryWrapper.tsx

```diff
@@ -301,7 +303,9 @@ export default class QueryWrapper extends React.Component<
       })
       .catch(error => {
         console.error('Failed to get data ', error)
-        this.setState(error)
+        this.setState({
+          error,
```

Comment:
Specifically, this error was not shown in the UI (this is the case where the query was re-executed and encountered an error in the service call).